dropFastRefreshModule.sh

```diff
@@ -9,7 +9,7 @@
 
 
 . ./module_set_variables.sh
-export LOG_FILE=/tmp/dropFastRefreshModule_`date +%Y%m%d-%H%M`.log
+export LOG_FILE=./dropFastRefreshModule_`date +%Y%m%d-%H%M`.log
```

Comment:
Chose to change this to be inside the `pg-mv-refresh` folder rather than the root `/tmp/` directory, seemed more appropriate